<launch>

    <node pkg="traversability_analysis" type="online_local_map_node" name="online_local_map_node" output="log">
        <remap from="/registered_scan" to="/registered_point_cloud"/>
        <remap from="/odometry" to="/laser_odom_init"/>

        <param name="frame_id" value="map"/>
        <param name="local_max_range" value="30"/>
        <param name="map_voxel_size" value="0.1"/>

    </node>

    <node pkg="traversability_analysis" type="traversability_analysis_node_3d" name="traversability_analysis_node" output="log" required="true">
        <remap from="/odometry" to="/laser_odom_init"/>
        <remap from="/static_point_cloud" to="/online_local_map_node/local_map"/>

        <param name="scan_voxel_size" value="0.1"/>
        <param name="vehicle_height" value="0.52"/>
        <param name="lower_bound_z" value="-0.55"/>
        <param name="upper_bound_z" value="0.0"/>
        <param name="dis_ratio_z" value="0.27"/>
        <param name="lower_z" value="0.3"/>

        <param name="grid_size" value="0.5"/>
        <param name="grid_width" value="60"/>
        <param name="min_grid_point_num" value="1"/>

    </node>

</launch>
